### PR TITLE
Adding Spanner STRUCT param samples

### PR DESCRIPTION
**Pull request by jabubake**
- [x] Tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

### CODE REVIEW

**Comment by jabubake** [comment]
@jsimonweb FYI

File: samples/struct.js [hunk 1]
```diff
@@ -0,0 +1,431 @@
...
+  // Creates a client
+  const spanner = new Spanner({
+    projectId: projectId,
+  });
+
+  // Gets a reference to a Cloud Spanner instance and database
+  const instance = spanner.instance(instanceId);
+  const database = instance.database(databaseId);
+
+  // Instantiate Spanner table objects
```
Comment (fhinkel) [inline samples/struct.js]: Nit: `Instantiates`? I personally prefer it without the `s`, but it should be consistent in one file.

File: samples/struct.js [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+  const singersTable = database.table('Singers');
+
+  // Inserts rows into the Singers table
+  // Note: Cloud Spanner interprets Node.js numbers as FLOAT64s, so
```
Comment (fhinkel) [inline samples/struct.js]: Suggestion: 

`Note: Cloud Spanner interprets JavaScript numbers as FLOAT64s. Use strings as in this examples if you need INT64s.`

Definitely write JavaScript numbers, not Node numbers.

File: samples/struct.js [hunk 3]
```diff
@@ -0,0 +1,431 @@
...
+        FirstName: 'Benjamin',
+        LastName: 'Martinez',
+      },
+      {
+        SingerId: '9',
+        FirstName: 'Hannah',
+        LastName: 'Harris',
+      },
+    ])
+    .then(() => {
```
Comment (fhinkel) [inline samples/struct.js]: You could shorten those to 
`.then( () => console.log('Inserted data.'))

File: samples/struct.js [hunk 4]
```diff
@@ -0,0 +1,431 @@
...
+      console.log('Inserted data.');
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    })
+    .then(() => {
```
Comment (fhinkel) [inline samples/struct.js]: You could shorten to 

`.then( () => database.close())`

If the function body is a single statement, return is implicit.

Comment (jabubake) [reply samples/struct.js]: Done

File: samples/struct.js (hunk 3 above)
Comment (jabubake) [reply samples/struct.js]: Done

File: samples/struct.js (hunk 2 above)
Comment (jabubake) [reply samples/struct.js]: Done

File: samples/struct.js (hunk 1 above)
Comment (jabubake) [reply samples/struct.js]: Done

**Comment by jabubake** [comment]
@fhinkel looks like the appveyor build is still pending for the last hour.
Otherwise, the PR is ready to merge.

**Pull request merged**